Makefile

```diff
@@ -95,3 +95,10 @@ gpu-dev: prereqs ## Runs the site as a developer; including live reload support
 .PHONY: debug
 debug: prereqs ## Runs the site as a developer with full debugging; including live reload support and installation of git hooks
 	./node_modules/.bin/supervisor -w app.js,lib,etc/config,static/tsconfig.json -e 'js|ts|node|properties|yaml' -n exit --inspect 9229 --exec $(NODE) $(NODE_ARGS) -- -r esm -r ts-node/register ./app.js --debug $(EXTRA_ARGS)
+
+.PHONY: asm-docs asm-docs-dependencies
+asm-docs: asm-docs-dependencies
+	$(MAKE) -C etc/scripts/docenizers
+
+asm-docs-dependencies:
+	pip3 install beautifulsoup4 pdfminer.six
```

Comment:
Is this taboo to do in a makefile?

Reply:
To me: yes - sorry! I think we probably should (but maybe not in this PR) make a virtual environment etc like we do in `infra`

Reply:
Thanks! Would it be better to add some docs somewhere saying "To re-generate the asm docs run `pip3 install beautifulsoup4 pdfminer.six && npm i && make asm-docs`" or to just add a message to the rule saying "If this errors, install xyz"?

Side note I'm realizing I need to fix some rules here, will do this evening!

Reply:
![image](https://user-images.githubusercontent.com/51220084/212580026-9bd8d3fa-3e24-4668-8b59-0104ee96fcb8.png)
I decided to do the second option